apps/web/src/pages/Communities/CommunityMembersPage/components/EditCommunityMemberDialog.tsx

```diff
@@ -118,26 +120,6 @@ const EditCommunityMemberDialog = ({
 
   return (
     <Dialog.Root open={isOpen} onOpenChange={setIsOpen}>
-      <Dialog.Trigger>
-        <Button
-          color="secondary"
-          aria-label={intl.formatMessage(
-            {
-              defaultMessage:
-                "Edit community roles of {userName} in {communityName}",
-              id: "WY3oiC",
-              description:
-                "Aria label for the dialog trigger to edit user community membership",
-            },
-            {
-              userName,
-              communityName: getLocalizedName(community.name, intl),
-            },
-          )}
-          icon={PencilSquareIcon}
-          mode="icon_only"
-        />
-      </Dialog.Trigger>
```

Comment:
Decoupling these dialogs from their trigger is going to cause the focus issue that popped up in the talent request cards. 

Thankfully, there are [two examples of how to accomplish this](https://codesandbox.io/p/sandbox/dropdownmenu-dialog-items-r9sq1q) with keeping the focus management.

Reply:
OK, I think I got it!
[fix dialog triggers](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12005/commits/1b873a4b1ff0505b35090e9334b9aa76e58cb441)

Reply:
Nice! That works so well :ok_hand: